version_test.go

```diff
@@ -0,0 +1,18 @@
+package cdk
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetVersion(t *testing.T) {
+	data := GetVersion()
+	require.True(t, len(data.Version) > 0)
+	require.True(t, len(data.GitRev) > 0)
+	require.True(t, len(data.GitBranch) > 0)
+	require.True(t, len(data.BuildDate) > 0)
+	require.True(t, len(data.GoVersion) > 0)
+	require.True(t, len(data.OS) > 0)
+	require.True(t, len(data.Arch) > 0)
+}
```

Comment:
Nitpick, maybe slightly more verbose:

```suggestion
	require.NotEmpty(t, data.Version)
	require.NotEmpty(t, data.GitRev)
	require.NotEmpty(t, data.GitBranch)
	require.NotEmpty(t, data.BuildDate)
	require.NotEmpty(t, data.GoVersion)
	require.NotEmpty(t, data.OS)
	require.NotEmpty(t, data.Arch)
```